src/applications/lgy/coe/form/tests/fixtures/mocks/document-list.json

```diff
@@ -0,0 +1,22 @@
+{
+  "data": {
+    "attributes": [
+      {
+        "id": 23215740,
+        "documentType": "54",
+        "createDate": 1644869501000,
+        "description": "Example 1",
+        "mimeType": "IssueCOE1.pdf",
+        "title": "Example 1"
+      },
+      {
+        "id": 23215741,
+        "documentType": "54",
+        "createDate": 1646092800000,
+        "description": "Example 2",
+        "mimeType": "IssueCOE2.pdf",
```

Comment:
Awaiting LGY response on this:
- `documentType` is expected to contain the file extension, e.g. `pdf`, but the mock data shows otherwise
- `mimeType` could be used to extract and show the file extension, but given that it's named `mimeType`, I would expect it to contain `application/pdf`